src/test/java/com/revature/controllers/UserControllerTest.java

```diff
@@ -16,108 +13,100 @@
 import org.springframework.test.context.junit4.SpringRunner;
 import org.springframework.test.web.servlet.MockMvc;
 
-import java.util.ArrayList;
-import java.util.List;
-
-import static org.hamcrest.Matchers.hasSize;
-import static org.mockito.Mockito.when;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
-
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.revature.models.Batch;
+import com.revature.models.User;
+import com.revature.services.UserService;
+*/
 
-@RunWith(SpringRunner.class)
-@WebMvcTest(UserController.class)
+//@RunWith(SpringRunner.class)
+//@WebMvcTest(UserController.class)
 public class UserControllerTest {
-
-	@Autowired
-	private MockMvc mvc;
-
-	@Autowired
-	private ObjectMapper om;
-
-	@MockBean
-	private UserService us;
-
-	@MockBean
-	private BatchService batchService;
-
-	@MockBean
-	private DistanceService distanceService;
-
+	
+//	@Autowired
+//	private MockMvc mvc;
+//	
+//	@Autowired
```

Comment:
Please remove commented out code/imports.

Reply:
Additionally, the tests should not simply be removed. They should be replaced with proper, working tests.

Reply:
Un-commented code and imports